src/util/std_expr.h

```diff
@@ -4273,8 +4273,8 @@ class ieee_float_op_exprt:public exprt
     const exprt &_lhs,
     const irep_idt &_id,
     const exprt &_rhs,
-    const exprt &_rm):
-    exprt(_id)
+    const exprt &_rm)
+    : exprt(_id, _lhs.type())
```

Comment:
How do we now this is the right type? For instance I would expect `f < 0.1` to be of type bool even if lhs is a float

Reply:
This expression is for operations with rounding mode; the result is always a float.

Reply:
Adding that to the documentation would be nice.

Reply:
done